cloud-haven-react-app/src/App.css

```diff
@@ -7,7 +7,7 @@
     /* red */
     --notif-color: #C33C54;
     /* sky blue */
-    --tab-color: #8EE3EF;
+    --tab-color: #BCEAFF;
```

Comment:
I'm not sure about this change. I'm not sure if we had the color scheme before we created the figma. We might need to talk to the group about this one.

![image](https://user-images.githubusercontent.com/23527660/110161676-86857800-7da2-11eb-9c20-5a78a7c2443b.png)

Reply:
We changed it back to the color the tabs were in my original pull request that we approved and merged. Also the color we changed it to is based directly from the Figma design which we also all approved. I think it makes more sense now to base our front-end work off the Figma rather than the color scheme which we never really said had to be our only colors used.